pkg/controllers/provisioning/provisioner_test.go

```diff
@@ -537,6 +536,78 @@ var _ = Describe("Provisioning", func() {
 			Expect(*allocatable.Cpu()).To(Equal(resource.MustParse("4")))
 			Expect(*allocatable.Memory()).To(Equal(resource.MustParse("4Gi")))
 		})
+		It("should account for daemonset spec affinity ", func() {
+			ExpectCleanedUp(ctx, env.Client)
+			cluster.Reset()
```

Comment:
This call and L540 get called during the AfterEach section for each Ginkgo test so we shouldn't need to execute them here.

Reply:
yes i forgot to clean them up as i was debugging the unit test setup. thanks for pointing out